docs/dictionary/keyword/menu.lcdoc

```diff
@@ -26,49 +26,62 @@ If a button's <style> is set to "menu", its <menuMode> <property> determines how
-  &lt;char&gt;                                 (Command key + specified char)
-  &lt;keyname&gt;                         (Named key without modifiers)
-  &lt;modifiers&gt; &lt;char&gt;           (Specified modifiers + character)
-  &lt;modifiers&gt; &lt;keyname&gt;   (Specified modifiers + named key)
-  
-Here &lt;modifiers&gt; is either:
-  1) a space separated list of:
-        'command' or 'cmd', 'control' or 'ctrl', 'option' or 'opt', 'alt', 'shift'
-        in this case the keyname/char should be separated from the list by a space
-  2) a sequence of characters:
-  ^        (Command)
-  @       (Shift)
-  #        (Option)
-  %       (Control)
-           in this case no space between the sequence and keyname/char is required.
+
+    [<flags>] <label> ['/' <accelerator> ['|' <tag>]]
+
+Where `<flags>` may include:
+
+* `!c`, `!n`, `!r`: the menu item has respectively, a check, no check, or a selected radio button
+* `(`: the menu item is disabled
+* `-`: the menu item is replaced with a menu separator
+* multiple tab characters: the number of tabs specifes the depth of the menu item; use this to create submenus
+
+The `<accelerator>` can be specified as one of:
+
+* `<char>`: Command key + specified char
+* `<keyname>`: Named key without modifiers
+* `<modifiers> <char>`: Specified modifiers + character
+* `<modifiers> <keyname>`: Specified modifiers + named key
+
+In the `<accelerator>`, `<modifiers>` is either:
+
+* a space separated list of: 'command' or 'cmd', 'control' or 'ctrl', 'option' or 'opt', 'alt', 'shift'; in this case the keyname/char should be separated from the list by a space
+* a sequence of characters, including:
+  * `^`: Command
+  * `@`: Shift
+  * `#`: Option
+  * `%`: Control
+
+  In this case no space between the sequence and keyname/char is required
 
 The following named keys are supported:
-        F1, F2, F3, F4, ... F15,
-        Left, Up, Right, Down,
-        Backspace, Delete,
-        Tab, Space,
-        Return, Enter,
-        Home, End, Escape,
-        PgUp, PgDown,
-        Insert (not on Mac OS X)
- 
+
+* F1, F2, F3, F4, ... F15,
+* Left, Up, Right, Down,
+* Backspace, Delete,
+* Tab, Space,
+* Return, Enter,
+* Home, End, Escape,
+* PgUp, PgDown,
+* Insert (not on Mac OS X)
+
 On Mac OS X, these keys will be replaced with the appropriate system standard glyph.
 
-A tag can be specified as a '|' character followed by a string of ascii characters.  If such a tag is present then that tag will take the place of the menu label as the parameter to the <menuPick message>.
-A tag will only be recognised following &lt;accelerator&gt;.  To specify a menu tag without an accelerator shortcut, an empty &lt;accelerator&gt; can be specified using '/|' followed by the tag text.
+A tag can be specified as a `|` character followed by a string of ASCII characters.  If such a tag is present then that tag will take the place of the menu label as the parameter to the <menuPick message>.
```

Comment:
The `<menuPick message>` here should be `<menuPick> message` (and the ref changed in the references)